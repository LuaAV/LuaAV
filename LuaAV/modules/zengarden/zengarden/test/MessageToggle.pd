#N canvas 746 271 450 300 10;
#X obj 63 19 loadbang;
#X obj 133 19 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 98 150 tgl 15 0 empty empty empty 17 7 0 10 -262144 -1 -1 1
1;
#X msg 204 105 33;
#X msg 168 105 0;
#X obj 98 179 print;
#X msg 120 105 -12.9;
#X msg 75 105 set 1;
#X obj 62 55 t b b b b b b b b b b;
#X msg 32 105 set 0;
#X connect 0 0 8 0;
#X connect 1 0 8 0;
#X connect 2 0 5 0;
#X connect 3 0 2 0;
#X connect 4 0 2 0;
#X connect 6 0 2 0;
#X connect 7 0 2 0;
#X connect 8 0 2 0;
#X connect 8 1 9 0;
#X connect 8 2 2 0;
#X connect 8 3 7 0;
#X connect 8 4 2 0;
#X connect 8 5 6 0;
#X connect 8 6 2 0;
#X connect 8 7 4 0;
#X connect 8 8 2 0;
#X connect 8 9 3 0;
#X connect 9 0 2 0;
