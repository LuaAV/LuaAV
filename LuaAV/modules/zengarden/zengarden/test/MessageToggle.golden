[@ 0.000ms] print: 33
[@ 0.000ms] print: 0
[@ 0.000ms] print: 0
[@ 0.000ms] print: 33
[@ 0.000ms] print: -12.9
[@ 0.000ms] print: 0
[@ 0.000ms] print: 0
[@ 0.000ms] print: 1